CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
 # Git LFS Changelog
 
+## 1.3.0 (21 July 2016)
+
+### Features
+
+* use proxy from git config  #1358 (@technoweenie)
```

Comment:
Gotcha in https://github.com/github/git-lfs/pull/1388/commits/afa77420f25e08f9df01b47db66e3e5b5a6b570f!